android/assets/jsons/translationsByLanguage/Traditional_Chinese.properties

```
@@ -2,33 +2,35 @@
 # Tutorial tasks
 
  # Requires translation!
-Move a unit!\nClick on a unit > Click on a destination > Click the arrow popup = 
+Move a unit!\nClick on a unit > Click on a destination > Click the arrow popup = 移動你的人民!\點擊一個單位 > 點擊其目的地 > 點擊確認箭頭前進
```

```suggestion
Move a unit!\nClick on a unit > Click on a destination > Click the arrow popup = 移動你的人民!\n點擊一個單位 > 點擊其目的地 > 點擊確認箭頭前進
```
(added missing n to \n)